packages/components/src/internal/components/assay/BatchPropertiesPanel.tsx

```diff
@@ -39,6 +40,7 @@ export const BatchPropertiesPanel: FC<AssayPropertiesPanelProps> = memo(props =>
                         fieldValues={model.batchProperties.toObject()}
                         queryColumns={model.batchColumns}
                         renderFileInputs
+                        containerFilter={Query.containerFilter.currentPlusProjectAndShared}
```

Comment:
When adding container filters to base components I've tried to be diligent about not marking explicit container filters on subcomponents and rather let the application level declarations dictate the functionality. I realize in this case it may be always this way (and it looks like we've sprayed this kind of thing around quite a bit for samples already in `@labkey/components`).

That said, this is also the intent of `getContainerFilterForLookups()` specifically in the scenario where projects are supported.

This same idea applies to other usages of container filters directly in this PR.

Reply:
Done.